mypy/newsemanal/semanal.py

```diff
@@ -2531,7 +2575,12 @@ def analyze_types(self, items: List[Expression]) -> List[Type]:
         result = []  # type: List[Type]
         for node in items:
             try:
-                result.append(self.anal_type(expr_to_unanalyzed_type(node)))
+                analyzed = self.anal_type(expr_to_unanalyzed_type(node))
+                if analyzed is not None:
+                    result.append(analyzed)
+                else:
+                    # TODO: Is this the right thing to do?
+                    result.append(AnyType(TypeOfAny.from_error))
```

Comment:
Should we just return `Optional[List[Type]]` from this function instead?

Reply:
Maybe, it's unclear at this point. We can get back to this later.